scripts/sysman/engineActivity.yml

```diff
@@ -0,0 +1,73 @@
+#
+# Copyright (C) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+#
+# See YaML.md for syntax definition
+#
+--- #--------------------------------------------------------------------------
+type: header
+desc: "Intel $OneApi Level-Zero Sysman Extension APIs for Engine Activity"
+version: "1.7"
+--- #--------------------------------------------------------------------------
+type: macro
+desc: "Engine Activity Extension Name"
+version: "1.7"
+name: $S_ENGINE_ACTIVITY_EXT_NAME
+value: '"$XS_extension_engine_activity"'
+--- #--------------------------------------------------------------------------
+type: enum
+desc: "Engine Activity Extension Version(s)"
+version: "1.7"
+name: $s_engine_activity_ext_version_t
+etors:
+    - name: "1_0"
+      value: "$X_MAKE_VERSION( 1, 0 )"
+      desc: "version 1.0"
+--- #--------------------------------------------------------------------------
+type: struct
+desc: "Extension properties related to Engine Groups"
+version: "1.7"
+class: $sEngine
+name: $s_engine_ext_properties_t
+base: $s_base_properties_t
+members:
+    - type: uint32_t
+      name: "countOfVirtualFunctionInstance"
+      desc: |
+            [out] Number of Virtual Function(VF) instances associated with engine to monitor the global utilization of hardware across all Virtual Function from a Physical Function (PF) instance.
+            These global and VF-by-VF views should provide engine group and individual engine level granularity.
```

Comment:
global includes PF usage as well right ? 
So should we mention only "VF views" here.

Reply:
yeah good point, this count represents number of VFs actively using particular resource. so I can remove global wording. does that seem fine @sanilkumar0 ?

Reply:
fixed